src/app/shell/personal-cabinet/provider/create-workshop/create-description-form/create-description-form.component.html

```diff
@@ -3,12 +3,16 @@
   <app-image-form-control [imgMaxAmount]="10" [label]="'Фото'" formControlName="image">
   </app-image-form-control>
 
-  <label class="step-label"> Опис <span class="step-required">*</span></label>
+  <div class="step-label-combined">
+    <label class="step-label" > Опис <span class="step-required">*</span></label>
+    <label class="step-letter-count" fxLayout='row' fxLayoutAlign='end'> {{letterCount.length}}/<span>{{500 -
+        letterCount.length}}</span></label>
+  </div>
+
   <mat-form-field>
     <textarea matInput class="step-textarea" placeholder="Максимум 500 символів" formControlName="description"
-      required></textarea>
+      [(ngModel)]='letterCount' maxlength="500" required></textarea>
```

Comment:
not sure that it is a good idea to use ngModel with reactive forms. You can subscribe on valueChanges of form control and update counter